insolar/jet/dbstore.go

```diff
@@ -87,7 +87,7 @@ func (s *DBStore) Clone(ctx context.Context, from, to insolar.PulseNumber) error
 	defer s.Unlock()
 
 	tree := s.get(from)
-	newTree := tree.Clone(false)
+	newTree := tree.Clone(true)
```

Comment:
This is not consistant with In-memory version. If we need to keep cloned tree, let's pass flag as argument for both versions.

Reply:
Added flag in both versions